db/schema.rb

```diff
@@ -134,6 +134,15 @@
 
   add_index "translations", ["content_item_id", "locale"], name: "index_translations_on_content_item_id_and_locale", using: :btree
 
+  create_table "unpublishings", force: :cascade do |t|
+    t.integer  "content_item_id", null: false
```

Comment:
Should we add an index here? Perhaps a composite of `content_item_id, type` as well?